packages/workers-playground/src/QuickEditor/VSCodeEditor.tsx

```diff
@@ -15,7 +15,7 @@ function stripSlashPrefix(path: string) {
 	return path[0] === "/" ? path.slice(1) : path;
 }
 
-const quickEditHost = "https://quick-edit.devprod.cloudflare.dev";
+const quickEditHost = "https://e98a7604.quick-edit-workers.devprod.cloudflare.dev";
```

Comment:
Maybe add a comment about why you use a preview (I think?) - at least in the description of the PR